docs/source/upgrade/upgrade-guide-from-monitoring-3.x-to-monitoring-4.y.rst

```diff
@@ -99,8 +99,7 @@ We assume that you are using external volume to store the metrics data.
 Backup
 ^^^^^^
 
-We suggest to copy the Prometheus external directory first and use the copy as the data directory for the new monitoring stack.
-Newer Monitoring stack uses newer Promethues versions, and keeping a backup of the prometheus dir would allow you to rollback.
+A copy of the Prometheus external directory should be made first and used as the data directory for the new monitoring stack. The new monitoring stack uses newer versions of Prometheus and keeping a backup would enable you to rollback to a previous version of Prometheus.
```

Comment:
why move to passive voice? I find the original version clearer.
You switch to singular, but we are talking about versions (i.e. 3.x and 4.y)

Reply:
To avoid the passive voice, we can use the following paragraph:

We suggest to make a copy of Prometheus' external directory and use it as the data directory for the new monitoring stack. The new monitoring stack uses newer versions of Prometheus and keeping a backup would enable you to rollback to a previous version of Prometheus.